ibis/expr/types/core.py

```diff
@@ -140,7 +140,17 @@ def __bool__(self) -> bool:
     __nonzero__ = __bool__
 
     def get_name(self):
-        """Return the name of this expression."""
+        """
+    Return the (possibly unqualified) name of the table.
```

Comment:
This isn't correct. This `get_name()` implementation is implement on the base class of scalar, column, *and* table expressions. Any discussion of tables should be on the corresponding subclass.

Also, the formatting here doesn't follow any of the surrounding docstring formatting.